src/Phoenix.vue

```diff
@@ -175,6 +184,16 @@ export default {
           this.$_updateNotifications()
         }, 30000)
       }
+    },
+    selectedLanguage: {
+      handler(language) {
+        if (language !== null) {
+          this.$language.current = language.listValue.values[0].stringValue
+        } else if (this.$language.defaultLanguage) {
```

Comment:
Is `defaultLanguage` always defined?

Reply:
Default language is configured as `navigator.language.substring(0, 2)`, so it should always be available. Still, I changed the code in a new commit, because it didn't allow to unset the setting. Please re-review. :-)

Reply:
I just wondered why was there the `else if`. But the current approach looks good 👍